tests/testthat/helper.R

```diff
@@ -0,0 +1,50 @@
+proj <- new.env(parent = emptyenv())
```

Comment:
Can't this entire file go into `test-use_beachmat.R`?

Reply:
Yeah, i think so